---
title: "The state of knowledge for marine invertebrate biodiversity in the continental US"
date: ' ' #'`r format(Sys.time())`'
output:
  bookdown::pdf_document2:
    includes:
    #  before_body: ../template/doc_prefix.tex
      in_header: preamble.tex
    keep_tex: yes
    latex_engine: xelatex
    number_sections: no
    toc: no
fontsize: 12pt
link-citations: yes
csl: nature.csl
bibliography: sok.bib
abstract: Museum collections have been recording our knowledge about the diversity and distribution of species for about 250 years. Until recently, this information was fragmented across institutions and difficult to access or synthesize. The digitization of biodiversity information is enabling the synthesis of collection data. Here we use publicly accessible digital biodiversity data to asses how well we know the marine invertebrate fauna of the continental United States. Globally, marine inverterbrates comprise about 75% of described marine species. We document `r format(n_distinct(combined_species$worms_valid_name), big.mark = ",")` species from the continental US waters in digital databases (OBIS and iDigBio), and explore spatial and temporal patterns of their distribution and collection. Databases do well at the fauna level, capturing ~xxx% of the species diversity. However, sampling bias obscure broad diversity patterns, such as differences between coasts and across latitude. Our knowledge about species richness and geographic range is limited by sampling effort, with `r prop_10_specimens_species`% of species known from $\leq$ 10 samples. While recent developments in high-throughput sequencing promise to greatly speed up documentation of marine biodiversity, interpretation of sequences requires anchoring results anchoring results to identified vouchers. DNA barcode sequences (COI) are presently available for only `r format_output(prop_spp_barcoded$p_barcoded * 100)`% of the US marine invertebrates, and many of these are from samples taken outside the US. Over a third of the barcoded species are complexes of of multiple cryptic lineages, indicating that the fauna is substantially richer than the above number indicate and needs to be taxonomically reevaluated as a whole. Substantially increased sampling, coordinated at the national scale, is needed to provide a meaningful baseline for marine biodiversity and for metabarcoding and metagenomics studies. This data will allow the detection of the impacts of climate change and other anthropogenic strains on marine biodiversity. Advances in computing, information systems, and sequencing technologies, have set the stage for a broad understanding of the dynamics of marine biodiversity, but this potentially is limited by the availablility of current samples, digitization, and taxonomic effort.
---

```{r setup, echo=FALSE, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
```


## Introduction

With increasing impact of human activities on the biosphere, we need efficient and accurate approaches to document changes in species diversity, distribution, and community composition. <!-- add why? -->  While terrestrial ecosystems have received considerable attention, the baseline of healthy and resilient marine ecosystems are not well understood [@Worm2006], and modeling the impact of human activities has been limited to a few well-studied taxonomic groups [@Tittensor2010]. Marine biodiversity remains poorly understood, with even the global richness of macrobiota poorly constrained[@Mora2011,@Appeltans2012,@Poore2015], and most species are known from few records and studies [@Bouchet2002]. Constrained by access to habitats and limited biodiversity infrastructure, coordinated, broad marine ecosyste monitoring initiatives have been slow to develop [@Duffy2013,@Borja2016] (but see for instance MarineGEO [@Duffy2016marinegeo], MBON, and PISCO), making it difficult to monitor changes in species distributions and community composition. Climate change and other human impacts are leading to rapid changes in marine ecosystems, and there is a pressing need to improve our knowledge of baseline marine systems and species diversity before they are further altered.

Biodiversity is fundamental to ecosystem function, but documenting and tracking biodiversity is laden with taxonomic and sampling challenges. For macrobiota, species occurrence has generally been documented through visual identification of live or captured specimens. Reliably-identified samples can then be used to generate molecular identification tools, such as DNA barcodes, opening additional methods for species documentation.

Species occurrence data is the basis for understanding species diversity and distributions, and is essential for monitoring community composition and ecosystem function. When species distributions are sufficiently well-sampled, occurrence data can also be used for environmental niche modelling to predict geographic and bathymetric distributions, community composition, and response of species and communities to changing environmental conditions. In well-sampled systems, like land plants, these can yield detailed prediction of future ecosystems (xxxx).

Until recently, our knowledge of biodiversity was scattered among heterogeneous sources (e.g., natural history collections, publications, expert knowledge) that are difficult to access and synthesize. The development of public databases for specimen records (e.g., iDigBio [@Page2015]), species occurrences (e.g., OBIS [@Zhang2002]), genetic data (e.g., GenBank [@Benson2013], BOLD [@Ratnasingham2007a]), and registers of species names (e.g., WoRMS [@Costello2013b]) have revolutionized how biodiversity research by allowing ready access to aggregated data to answer broad questions. However, few studies have sought to synthesize existing knowledge by combining information from these databases. 

The goal of this study is to evaluate the extent of, and gaps in, our digital knowledge of species occurrences of marine invertebrates in the marine waters of the continental United States, one of the best documented areas in the world ocean. We aggregate publicly available data within the limits of the exclusive economic zone (EEZ) of the US, as well as the Gulf of Mexico (GOM) and the Pacific Northwest (PNW). We show that while public databases are capturing a significant portion of our knowledge about species richness, most species are known from only a few specimens and locations. We emphasize that our understanding of marine biodiversity, and the impact of human activities on biological communities require not only a more accurate knowledge of the species that inhabit the oceans, but also more comprehensive data on species distributions and abundances. These goals can only be achieved with additional cost-effective sampling and a commitment to developing and maintaining the taxonomic expertise who can identify and interpret the material collected.

We focus on the continental US because the marine biodiversity is extensively documented as a result of a long history of studies by governmental, academic, and private institutions [@Fautin2010]. The continental US includes a wide range of marine ecosystems from the Gulf of Mexico and Florida Keys, to the Gulf of Maine on the East Coast; and from arid Southern California to the shores of the Pacific Northwest on the West coast. With estuaries, bays, sandy beaches, rocky shores, and coral reefs, the US coastlines harbors many biota. While the continental shelf extends for several hundred kilometers on the East coast, it is narrow on the West coast. This diversity of habitats and bathymetric profiles, combined with efforts at the national level to make data publicly available, make this region especially suitable to evaluate how much we know about marine biodiversity using biodiversity repositories. Additionally, apart from the syntheses that emerged from the Census of Marine Life efforts [@Costello2010,@Fautin2010], no study has yet attempted to synthesize the current state of knowledge of marine biodiversity across the continental US, and this study is the first using data from public databases.


## Results and Discussion

<!---- Research attempting to understand how much we know about marine biodiversity has focused on estimating the total number of species that inhabits the oceans, and which fraction of this diversity has been named [@Appeltans2012,@Costello2013]. Undoubtedly, the development of molecular methods to investigate species limits has reinvigorated taxonomy and contributed to elucidating cryptic species complexes [@Knowlton2000]. Thus, it has been argued that molecular data can speed up the enterprise of discovering and naming species [@Blaxter2016]. As species remain the most convenient unit to measure biological diversity, understanding how much has been done, how much remains to be done, and how we can speed up the description of species diversity is crucial. However, to understand the role of these species in their environment, we need to document their ecology [@Cernansky2017]. For instance, to be able to predict how their geographic ranges will be affected by global change, we need detailed information about their environmental requirements. Occurrence data for marine species could be used for environmental niche modeling of their geographic and bathymetric distributions but the number of species -- and therefore the proportion of all species -- for which we have enough data to do so is unknown. -->


### What do we know about marine invertebrates in the US?

We retrieved `r format(n_distinct(combined_species$worms_valid_name), big.mark = ",")` species (`r n_distinct(na.omit(combined_species$worms_phylum))` phyla) of marine invertebrates in the continental US EEZ by combining  `r format(nrow(idigbio_us_inverts_records_species), big.mark=",")` museum records from iDigBio (`r format(n_distinct(idigbio_us_inverts_records_species$worms_valid_name), big.mark = ",")` species, `r n_distinct(na.omit(idigbio_us_inverts_records_species$worms_phylum))` phyla), and `r format(nrow(obis_us_inverts_records_species), big.mark=",")` occurence records from OBIS (`r format(n_distinct(obis_us_inverts_records_species$worms_valid_name), big.mark=",")` species,
`r n_distinct(na.omit(obis_us_inverts_records_species$worms_phylum))` phyla). 


```{r kingdom-stats}
get_invert_prop <- function(db, what = c("invert", "vert")) {
    what <- match.arg(what)
    string_what <- c("invert" = "Animalia - Invertebrates",
                     "vert" = "Animalia - Vertebrates")
    what <- string_what[what]
    format_output(kingdom_diversity[kingdom_diversity$database == db &
                                    kingdom_diversity$sub_kingdom == what,
                                    "prop_spp"] * 100)
}

total_invert_prop <- get_invert_prop("worms", "invert")
idigbio_invert_prop <- get_invert_prop("idigbio", "invert")
obis_invert_prop <- get_invert_prop("obis", "invert")

total_vert_prop <- get_invert_prop("worms", "vert")
idigbio_vert_prop <- get_invert_prop("idigbio", "vert")
obis_vert_prop <- get_invert_prop("obis", "vert")

median_obis_vert <- kingdom_stats %>%
    filter(database=="obis" & sub_kingdom == "animalia - vertebrates") %>%
    pull("median_sample")

median_obis_invert <- kingdom_stats %>%
    filter(database=="obis" & sub_kingdom == "animalia - invertebrates") %>%
    pull("median_sample")


median_idigbio_vert <- kingdom_stats %>%
    filter(database=="idigbio" & sub_kingdom == "animalia - vertebrates") %>%
    pull("median_sample")

median_idigbio_invert <- kingdom_stats %>%
    filter(database=="idigbio" & sub_kingdom == "animalia - invertebrates") %>%
    pull("median_sample")
```


```{r records-table, results='asis'}
xtbl_small <- function(x) {
    glue::glue("{{\\footnotesize {x} }}")
}
xtbl_bold <- function(x) {
    glue::glue("{{\\bfseries {x} }}")
}
print.xtableList(institution_contributions, colnames.format = "multiple",
                 booktabs = TRUE, include.rownames = FALSE,
                 sanitize.subheadings.function = xtbl_bold, 
                 sanitize.message.function = xtbl_small, 
                 table.placement = "!ht", comment = FALSE)
```


Marine invertebrates are underrepresented in iDigBio compared to other taxa. Marine invertebrates represent 11% of described global species diversity (202,000 [@worms-stats] out of 1.7 million species [catalog-of-life]), but only `r format_output(nrow(idigbio_us_inverts_records_species)/idigbio_stats$eez_no_fossil*100)`% of the US non-fossil occurrence records ((`r round(idigbio_stats$eez_no_fossil/1e6, 1)` million) in iDigBio. 


```{r kingdom-diversity-plot, fig.cap='Comparison of the proportions of the diversity represented in iDigBio and OBIS  with the global diversity (as reported by WoRMS) for each kingdom. Vertebrates are overrepresented, while other animal groups are underpresented. Total number of species for each category are indicated next to the bars.(\\#fig:kingdom-diversity-plot)'}
knitr::include_graphics("figures/plot_kingdom_diversity.pdf")
```

Invertebrates are also represented in the marine data. About `r total_invert_prop`% of described marine species are invertebrates. The proportional species diversity of invertebrates is captured relatively effectively among US marine EEZ records in OBIS (`r invert_props$prop_obis_invert_species`%) and iDigBio (`r invert_props$prop_idigbio_invert_species`%). However, invertebrate occurrence records are disproportionately low: `r invert_props$prop_obis_invert_samples`% in OBIS and `r invert_props$prop_idigbio_invert_samples`% in iDigbio. Marine vertebrate species have three times as many occurrence records on average in these databases than invertebrate species (median number of records per species for invertebrates `r median_idigbio_invert` and `r median_obis_invert`, _vs._ and `r median_idigbio_vert` and `r median_obis_vert` for vertebrates, for iDigBio and OBIS respectively).

To assess how well documented species diversity is captured digitally, we compared the species retrieved from iDigBio and OBIS against comprehensive checklists for molluscs [@turgeon1998] and crustaceans [@mclaughlin2005] of the United States. Of the `r format(itis_stats$moll_nspp, big.mark=",")` mollusk species included in the checklist, `r itis_stats$prop_moll_idig`% are in iDigBio and `r itis_stats$prop_moll_obis`% are in OBIS. Of the `r format(itis_stats$crust_nspp, big.mark=",")` crustacean species in the checklist, `r itis_stats$prop_crust_idig`% are in iDigBio and `r itis_stats$prop_crust_obis`% are in OBIS. Given that molluscs and crustaceans are among the most sampled taxa, and that they have received more taxonomic attention than other phyla (Fig.  \@ref(fig:idigbio-through-time)), these percentages offer a conservative estimate for the fraction of the fauna that is available digitally. Therefore, using a conservative estimate of a 60% digitization level of the fauna, we can estimate that there is at least 29,000 species of marine invertebrates recorded in US waters. The higher overlap between the comprehensive checklist for crustaceans in OBIS than in iDigBio is attributable to planktonic species that are better represented in OBIS data than in museum collections.

The faunistic list we assembled represent a subset of the species that have been digitized, because only records that are associated with geographic coordinates are included. All records published by OBIS have geographic coordinates associated with them. To estimate the proportion of species missed because they lacked geographic coordinate in iDigBio, we queried iDigBio using the comprehensive list for molluscs and crustaceans, and calculated the proportion of species with no iDigBio records within the US EEZ, but which had "United States" or "USA" in the country field. These species should occur within within the limits of the US EEZ but would not be returned in the queries used to assemble the faunistic list. We found that `r format_output(itis_stats$missing_geo_moll$prop_us_no_geo * 100)`% ($n=$ `r format(itis_stats$missing_geo_moll$n_spp, big.mark=",")`) of the species included in the comprehensive checklist for molluscs have records in iDigBio that place them in the United States based on the "Country" field, but do not have geographic coordinates associated with them that would ensures they fall within the limits of the EEZ. This percentage decreases to `r format_output(itis_stats$missing_geo_moll$prop_us_no_geo_no_obis * 100)`% once we exclude species that have informed geographic coordinates in OBIS. For crustaceans, these proportions are lower at `r format_output(itis_stats$missing_geo_crust$prop_us_no_geo * 100)`% and `r format_output(itis_stats$missing_geo_crust$prop_us_no_geo_no_obis * 100)`% ($n=$ `r format(itis_stats$missing_geo_crust$n_spp)`) respectively. This means ...



```{r local-richness}
count_spp <- . %>%
    dplyr::pull(worms_valid_name) %>%
    dplyr::n_distinct()

n_spp_gom <- combined_records %>%
    dplyr::filter(within_gom == TRUE) %>%
    count_spp

n_spp_east <- combined_records %>%
    dplyr::filter(is_east_coast == TRUE) %>%
    count_spp

n_spp_west <- combined_records %>%
    dplyr::filter(is_west_coast == TRUE) %>%
    count_spp


prop_idigbio_continental_shelf <- (
   mean(idigbio_us_inverts_records_all_with_depth$marmap_depth < 0 &
        idigbio_us_inverts_records_all_with_depth$marmap_depth > -200) * 100) %>%
   format_output()


prop_obis_continental_shelf <- (
    mean(obis_us_inverts_records_all_with_depth$marmap_depth < 0 &
         obis_us_inverts_records_all_with_depth$marmap_depth > -200) * 100) %>%
    format_output()
```


The Atlantic Ocean and the Gulf of Mexico are much better doocumented in these databases than the Pacific Ocean (`r format(sum(combined_records$is_east_coast), big.mark=",")`, `r format(sum(combined_records$within_gom), big.mark=",")`, and `r format(sum(combined_records$is_west_coast), big.mark=",")` records respectively). The highest species richness has been recorded from the Gulf of Mexico (`r format(n_spp_gom, big.mark=",")` species), followed by the Atlantic Ocean (`r format(n_spp_east, big.mark=",")` species), and the Pacific Ocean (`r format(n_spp_west, big.mark=",")` species). The higher species richness documented on the East coast partly reflects sampling bias, reflected in many more records, partly attributable to the presence of more and older research institutions, higher levels of digitization (Table \@ref(tab:records-table)), and wider continental shelf there. Most samples have been collected on the continental shelf (Fig. \@ref(fig:map-diversity-sampling)A), with `r prop_idigbio_continental_shelf`% (iDigBio) and `r prop_obis_continental_shelf`% (OBIS) of the records collected between $leq$200m.

Sampling bias is further reflected in the lack of clear latitude diversity gradient (well documented regionally e.g. [@macpherson2002,@roy2000]) and differences between the Pacific and the Atlantic coasts. On the Pacific coast, the recorded richness is lowest around 40° of latitude, whereas on the Atlantic coast, it is highest at the same latitude (Fig. \@ref(fig:map-diversity-sampling)B, Fig. \@ref(fig:latitudinal-gradient)B). These trends are strongly influenced by sampling effort (Fig. \@ref(fig:latitudinal-gradient)C). Notably, for the Pacific coast, there is a deficit in the number of samples and species between 40° and 45° of latitude corresponding to the Oregon coast. In contrast, a peak in species and records around 42° on the Atlantic coast may be attributed to the Woods Hole Oceanographic Institute. 

```{r map-diversity-sampling, fig.cap='Spatial distribution of the number of samples (A) and species richness (B) for marine invertebrates based on records retrieved from iDigBio and OBIS within the limits of the Exclusive Economic Zone for the continental USA (represented in light gray) in 0.2 x 0.2 degree cells. (\\#fig:map-diversity-sampling)'}
knitr::include_graphics("figures/combined_map_sampling.pdf")
```

```{r latitudinal-gradient, fig.cap='Latitudinal gradient in species richness for the Pacific and Atlantic oceans (Gulf of Mexico records excluded), based on inferred species richness (A) and recorded species richness (B). The patterns are strongly influenced by the sampling effort (C). Trend lines are rolling means with a window of size of 3. (\\#fig:latitudinal-gradient)'}
knitr::include_graphics("figures/gradient.pdf")
```


```{r correlation-sampling, fig.cap='Relationship between number of records and number of species per 0.2 x 0.2 degree cell in US EEZ (Fig. \\@ref(fig:map-diversity-sampling)). The black line represents the 1:1 relationship. (\\#fig:correlation-sampling)'}
knitr::include_graphics("figures/plot_sampling_effort.pdf")
```

<!-- The median number of cells occupied by species is `r median(rastercell_distribution$n_cell)`. -->

Further indication of sampling bias is a tight correlation between the number of samples and number of species per cell used in Fig. \@ref(fig:map-diversity-sampling), and `r format_output(stat_sampling_effort$p * 100)`% (n = `r format(stat_sampling_effort$n, big.mark=",")`) of the cells with $\leq$ 500 records have a number of species within 10% of the number of records (Fig. \@ref(fig:correlation-sampling)). For most of the US EEZ, the sampling effort is insufficient to capture the species diversity acurately.

Environmental conditions are among the primary determinants of species distributions and can thus be used to estimate the response of organisms to climate change. Predictive ecological models forecasting changes in distributions are trained on environmental conditions encountered by species in their current distributions. To obtain reliable results, models require at least 10, but ideally 30 or more occurrence records per species [@Wisz2008]. Yet at present, `r prop_10_specimens_species`% of species are represented by $\leq$ 10 records, and `r prop_30_specimens_species`%  by $\leq$ 30 records based on the combined records in iDigBio and OBIS. Underscoring the inadequacy of the sampling effort, `r prop_singleton_species`% of the species are known from a single record.

```{r idigbio-mms}
prop_mms_75_85 <- (idigbio_mms_summary_by_year %>%
                   dplyr::filter(year >= 1975 & year <= 1985) %>%
                   dplyr::pull(n) %>%
                   sum()) / sum(dplyr::pull(idigbio_mms_summary_by_year, n))
```


```{r idigbio-through-time, fig.cap='Cumulative number of samples (A) and species (B) of marine invertebrates recorded through time based on iDigBio records. Relationship between number of samples collected, and number of species recorded for the first time (C), each point represents numbers for a given year, dashed line shows the 1:1 and dotted line the 1:100 ratios for number of species recorded for the first time to the number of samples. (\\#fig:idigbio-through-time)'}
knitr::include_graphics("figures/idigbio_through_time.pdf")
```

Another indication of the vagaries and inadequacy of the sampling is shown in the temporal record of collected species and documented species (Fig. \@ref(fig:idigbio-through-time)). Two time periods stand out: the 1870-80's and the 1960-80's. The first represents intensive efforts to document the US marine biota during the formative stages of the major museums and research organizations focused on marinie life, while the second is in large part the result of the biodiversity surveys conducted under the auspices of the Bureau of Ocean Energy Management (BOEM; formerly part of Bureau of Land Management, BLM, and the Minerals Management Service, MMS). At least `r format(sum(idigbio_mms_summary_by_year$n), big.mark=",")` records (or `r format_output(sum(idigbio_mms_summary_by_year$n)/nrow(idigbio_us_inverts_records_all)*100)`% of all US marine invertebrates) in iDigBio were from these surveys, `r format_output(prop_mms_75_85 * 100)`% of them collected between 1975 and 1985. Of the `r idigbio_mms_records %>% filter(rank %in% c("Species", "Subspecies")) %>% summarize(n = n_distinct(worms_valid_name)) %>% format(big.mark=",")` species collected in these surveys, `r nrow(idigbio_mms_species_only)` have not been collected outside of these surveys.

Since the BLM-MMS surveys, the rate of collection and documentation of new records have both slowed. During the 1960-1970 ~XX records were added per year, averaging `r round(n_spp_comparison$n_spp_1970_1960/11)` species.year^-1^ and 
`r round(n_spp_comparison$n_spp_2017_2000/18)` species.year^-1^ since 2000.

<!--- Note also difference in evenness across phyla – mollusks much more even, annelids most jerky – reflects broad vs focused collecting efforts].  Note however that the number of new records per sample taken has not declined appreciably (X to Y) in this period -->

The strong correlations between (1) the number of samples collected and the species richness (Fig. \@ref(fig:correlation-sampling)), and (2) the number of specimens collected per year and the number of species recorded for the first time (Fig. \@ref(fig:idigbio-through-time)C), also suggest that much of the US marine biodiversity remains undocumented.

Further biases exist across phyla and body size. Large-bodied and easily accessible (e.g. dried specimens like shells, corals, echinoderms) specimens are over-represented in museum collections, compared with small less accessible organisms, such as the meiofauna. Of `r n_idigbio_phyla` animal phyla represented from US waters in iDigBio, `r records_rare_phyla$n_rare_phyla` have <500 records. While these phyla represent
`r records_rare_phyla$prop_spp_rare_phyla_collected`% (n = `r format(records_rare_phyla$total_spp_rare_phyla_collected, big.mark=",")` total species) of the global number of species <!-- source unclear -->, they make up `r records_rare_phyla$prop_recs_rare_phyla`% of the iDigBio records. This proportion rises to `r records_rare_phyla$prop_spp_common_phyla_collected`% (n = `r format(records_rare_phyla$total_spp_common_phyla_collected, big.mark=",")`) for the `r n_idigbio_phyla - records_rare_phyla$n_rare_phyla` other phyla. Yet, these undersampled taxa, especially those of small body size, are often the most abundant, and are likely to play an important role in the detection of changes in environmental quality[@Bik2012]. Additionally, the ecological implications of changes in the structure and composition of the communities of these small-bodied organisms, on higher trophic levels and ecosystems are largely unexplored.

The proportion of samples in museum collections that are identified to species decreases substantially toward the present, and shows a marked decline around the 1970's (Fig. \@ref(fig:identification-through-time)), demonstrating limiting taxonomic effort and expertise. 

```{r id_though_time, fig.cap='Proportion of marine invertebrate specimens collected through time identified at the species level. Trend lines are  smoothed rolling mean on a 3 year window. For all groups, we observe a decline in the proportion of specimens collected that are identified at the species level, suggesting a lack of expertise to identity precisely what is collected. (\\#fig:identification-through-time)'}
knitr::include_graphics("figures/idigbio_identification_level_through_time.pdf")
```


### Comparison with Regional Lists by Geography

We extracted `r format(nrow(gom_worms_species), big.mark=",")` and `r format(nrow(kozloff_worms_species), big.mark=",")` marine invertebrate species from checklists of the Gulf of Mexico [@Tunnell2009] and ofx the Pacific Northwest [@Kozloff1996] respectively. For the Gulf of Mexico, biodiversity databases included more species for the same geographic area than the one used by the authors of the checklist (iDigBio and OBIS combined: `r format(dplyr::n_distinct(combined_gom_records$worms_valid_name), big.mark=",")` species; iDigBio only: `r format(n_distinct(idigbio_gom_records$worms_valid_name), big.mark=",")` species, $n =$  `r format(nrow(idigbio_gom_records), big.mark=",")` records; OBIS only: `r format(n_distinct(obis_gom_records$worms_valid_name), big.mark=",")` species, $n=$ `r format(nrow(obis_gom_records), big.mark=",")` records). For the PNW, fewer species than included in the regional checklist were recovered from biodiversity databases (iDigBio and OBIS combined: `r format(dplyr::n_distinct(combined_pnw_records$worms_valid_name), big.mark=",")` species; iDigBio only: `r format(dplyr::n_distinct(idigbio_pnw_records$worms_valid_name), big.mark=",")`  species, $n=$ `r format(nrow(idigbio_pnw_records), big.mark=",")` records; OBIS only `r format(dplyr::n_distinct(obis_pnw_records$worms_valid_name), big.mark=",")` species, $n=$ `r format(nrow(obis_pnw_records), big.mark=",")` records). The combination of the species listed in these three sources leads to `r format(n_distinct(total_gom_records$worms_valid_name), big.mark=",")` species for the GOM and `r format(n_distinct(total_pnw_records$worms_valid_name), big.mark=",")` species for the PNW. Conversely, about XX% of the total species records for the Gulf of Mexico, and XX% for the Pacific Northwest are missing from the checklists and represented only in databases.

```{r compare_database_coverage, fig.cap='Overlap between the regional species list (list), records in iDigBio (idigbio), records in OBIS (obis), and records in BOLD (bold) for the Gulf of Mexico (left) and the Pacific Northwest (right). A higher proportion of species in the regional checklists are captured in databases for the Gulf of Mexico compared with the Pacific Northwest. This plot is generated using the UpSet technique[@Lex2014;@Conway2017], which uses a matrix-based layout to represent intersections of sets and their sizes. The rows represent the sets, the columns represent the intersections, and each set that is part of the intersection is marked with a black-filled circle in the cells underneath. Black lines connect the sets that are part of the intersections. (\\#fig:database-coverage)', echo=FALSE}
knitr::include_graphics("figures/database_coverage.pdf")
```

```{r}
set_size_gom <- function(...)
    set_size_with_checklist(..., d = remake::fetch("total_gom_species"))
set_size_pnw <- function(...)
    set_size_with_checklist(..., d = remake::fetch("total_pnw_species"))
```

A higher proportion of the species included in the regional list for the GOM are captured in biodiversity databases than for the PNW. While `r format_output(set_size_gom("checklist","idigbio","obis")/n_distinct(total_gom_records$worms_valid_name) * 100)`%  of the species included in the checklist for the GOM have records in at least one of the biodiversity database (Fig. \@ref(fig:database-coverage)A), only `r format_output(set_size_pnw("checklist","idigbio","obis")/n_distinct(total_pnw_records$worms_valid_name)*100)`% do for the PNW  (Fig. \@ref(fig:database-coverage)B). Additionally, a higher proportion of species occur in the biodiversity databases but are not included in the regional checklist for the GOM compared with the PNW (GOM: `r format_output(set_size_gom("checklist","idigbio","obis",include_checklist=FALSE) / n_distinct(total_gom_records$worms_valid_name)*100)`%, $n =$ `r format(n_distinct(total_gom_records$worms_valid_name), big.mark=",")`; PNW:
`r format_output( set_size_pnw("checklist","idigbio","obis",include_checklist=FALSE) / n_distinct(total_pnw_records$worms_valid_name)*100)`%, $n =$ `r format(n_distinct(total_pnw_records$worms_valid_name), big.mark=",")`).


```{r compare_database_overlap, fig.cap='Proportions of the total number of species by phylum that are absent from faunistic lists but are in iDigBio or OBIS ("Not in list"), and absent from iDigBio or OBIS but are in the faunistic lists ("Not in databases"). Many species, even in diverse taxonomic groups, known to occur in these regions have no digital records. The numbers represent the number of species. (\\#fig:database-overlap)'}
knitr::include_graphics("figures/database_overlap_percent.pdf")
```

The more diverse phyla contribute the most to the discrepancies between the regional species lists and the database records (Fig. \@ref(fig:database-overlap)). OBIS tends to be closer to the full record set, but iDigBio has more species for the Gulf of Mexico that are not included in the list across most phyla. Many species of mollusks, arthropods, and annelids, known to occur in the Gulf of Mexico or in the Pacific Northwest, have no digital records in these geographic areas. 

<!-- Regional faunistic lists assembled and curated by taxonomic experts provide a comprehensive and detailed account of species that are known to occur in a given geographic area. We assessed how faunistic lists assembled over decades by specialists compare with records in public databases for the same geographic area. -->


Several factors may contribute to species being included in the regional lists but not having records in biodiversity databases. The first category of biases is linked to the likelihood of having specimens for these species in museum collections. Rare species may only have been collected a few times, and the corresponding records have not been digitized yet. Species that are difficult to preserve because they are large, very small, or fragile, are underrepresented in collections. The other category of biases is associated with data quality for these records. Some of these species may not have been digitized, or lack geographic coordinates associated, and therefore would be missed by our queries. While it is difficult to assess biases associated with the likelihood of preservation of a species, we investigated how missing geographic coordinates affect the discrepancies between the checklists and the databases.

<!-- include bias in digitization -->

Missing geographic coordinates reduces the overlap between the list of species from the regional list and the databases. Using iDigBio, we found that for the `r format(not_really_in_database$idig_gom_n_spp_not_in_db, big.mark=",")` species that do not appear within the geographic range of the Gulf of Mexico but are on the list, `r format_output(not_really_in_database$idig_gom_p_spp_not_in_db*100)`% are completely absent from iDigBio, while of the `r format(not_really_in_database$idig_gom_n_spp_no_coords, big.mark=",")` species that do appear in iDigBio, `r format_output(not_really_in_database$idig_gom_p_spp_no_coords*100)`%  do not have geographic coordinates associated with them. Additional digitization and georeferencing of collections is needed to improve biodiversity knowledge at regional scales. For instance, one can expect that the digitization coverage for the Gulf of Mexico will greatly increase once records from the Florida Wildlife Research Institute collections (St Petersburg, FL) become integrated into biodiversity information systems. Similarly, only a small fraction of the marine invertebrates records from institutions located on the Pacific coast has been digitized, as indicated by the paucity of Pacific coast collections listed in Table \@ref(tab:records-table).

Two factors most likely contribute to explanations of why some species are in databases but are not included in the regional checklists: (1) the species have been documented for the first time since the publication of the checklist, (2) the species records were not accessible or known to the authors, or (3) the species records are erroneaous and were excluded by the compilers of the checklist.

Recently recorded species contribute modestly to this discrepancy for the Gulf of Mexico. Only `r sum(recently_collected$gom_in_dbs$min_year > 2001)` species (`r format_output( sum(recently_collected$gom_in_dbs$min_year > 2001)/n_distinct(total_gom_records$worms_valid_name) * 100)`% of the total species richness, `r format_output(sum(recently_collected$gom_in_dbs$min_year > 2001)/spp_not_in_lists$gom_not_in_list * 100)`% of the species not on the checklist) are only represented by collections made since the publication of the checklist. For the older Pacific Northwest checklist, this bias is larger with  `r sum(recently_collected$pnw_in_dbs$min_year > 1996)` species (`r format_output(sum(recently_collected$pnw_in_dbs$min_year > 1996)/n_distinct(total_pnw_records$worms_valid_name) * 100)`% of the total species richness,  `r format_output(sum(recently_collected$pnw_in_dbs$min_year > 1996)/spp_not_in_lists$pnw_not_in_list * 100)`% of the species not on the checklist) collected subsequent to publication.

The authors of these checklists had no or limited access to digital databases that have since been developed and this likely contributed to species being missed. 


<!-- GP says he moved this up, but I don't see it: Many of these records were difficult to discover before digitization which, for the most part, has occurred after the compilation of these checklists. We estimate that using biodiversity databases increases the number of species included in the checklist by `r format_output(spp_not_in_lists$gom_not_in_list/n_distinct(total_gom_species$worms_valid_name)*100)`% for the Gulf of Mexico, and `r format_output(spp_not_in_lists$pnw_not_in_list/n_distinct(total_pnw_species$worms_valid_name)*100)`% for the Pacific Northwest. -->



### Molecular data for the future of biodiversity discovery and monitoring

Since its introduction in the early 2000s, DNA barcoding has revitalized invertebrate taxonomy by facilitating species identification and delineation [@Meyer2005-plos]. More recently, advances in high-throughput sequencing technologies have enabled the parallel sequencing of thousands of specimens in environmental samples (eDNA) cost effectively [@Leray2015], making it possible to estimate diversity and identify taxa from species-rich samples at unprecedented scales. With decreasing sequencing costs, and the prospect of automating sample collection, these technologies are poised to revolutionize large-scale assessment and monitoring of biodiversity [@Bohan2017]. However, the utility of the data generated hinges on reliably identified DNA barcode sequences that can serve for matching and identification of eDNA transcripts [@Leray2015]. Thus insights from eDNA technologies depend on the level of completeness and accuracy of comparative, identified, species-level DNA barcode coverage.

The marine invertebrates of the US are genetically characterized at a relatively high level: `r format_output(prop_spp_barcoded$p_barcoded * 100)`% (n = `r format(nrow(dplyr::distinct(combined_species, worms_valid_name)), big.mark=",")`) of the digitized fauna has DNA barcodes available from specimens taken somewhere in the world. Coverage varies by phylum from `r format_output(stat_barcoding$min_eez_value * 100)`% (`r stat_barcoding$min_eez_phylum`) to `r format_output(stat_barcoding$max_eez_value * 100)`% (`r stat_barcoding$max_eez_phylum`) (Fig. \@ref(fig:proportion-barcoded)). The GOM has comparable DNA barcoding coverage to the US average, but the PNW is better barcoded with >30% species-level coverage across each of the five most diverse phyla. This is driven in part by the active Canadian barcoding efforts in British Columbia.  In 2012, it was estimated that, at a global scale, less than 3% of the marine species had been sequenced [@Appeltans2012], and that this number was even lower in some areas (e.g., 2.6% for the Antarctic [@Grant2009]).

The above estimates are based on current taxonomy, but many species as currently understood are complexes of multiple cryptic species [@Knowlton2000;@Meyer2005-evolution]. BOLD uses an algorithm to assess species-level genetic differentiation among samples, separating molecular operational units into different BINs (BOLD Index Numbers)[@Ratnasingham2013]. More than a third of the marine invertebrate species recorded from the US and represented in BOLD include multiple BINs (`r format_output(100 * stat_bold$p_spp_multi_bin)`% of `r format(stat_bold$n_spp_not_singleton, big.mark=",")` species represented by more than XX sequences). XX% of the fauna is only known from single barcodes at this stage. Given the relatively sparse DNA coverage among as well as within species, the actual level of crytpic differentiation is higher. Many of the BOLD sequences of US species are based on specimens collected outside the US. Thus both the diversity and species-level identity of the marine fauna needs to be comprehensively reevaluated.

Specimens identified under the same species but sampled from distant locations are especially prone to include cryptic species. While geographic information is missing for most records in BOLD (`r format_output(100*stat_bold$p_no_coords)`% of the `r format(stat_bold$n_total_records, big.mark=",")` BOLD records that match a species name in our checklist have no geographic coordinate information associated with them), of the `r format(bold_geography_stats$n_spp_with_records, big.mark=",")` marine invertebrates species from the US with BOLD records, only `r format_output(bold_geography_stats$p_within_eez * 100)`% have at least one record collected within the EEZ and `r format_output(bold_geography_stats$p_in_north_america * 100)`% list a North American country as the site of collection. It is crucial to ensure that accurate geographic information is provided when DNA barcodes are published, and that enough samples for each species are sequenced to capture their geographic range and  genetic diversity.

```{r proportion-barcoded, fig.cap='Proportion of species with publicly available DNA barcodes  known to occur the US EEZ (iDigBio and OBIS records), in the Gulf of Mexico (iDigBio, OBIS, and checklist[@Tunnell2009]), and in the Pacific Northwest (iDigBio, OBIS, and checklist[@Kozloff1996]). Numbers next to the bars indicate the number of species for which DNA barcodes are available. (\\#fig:proportion-barcoded)'}
knitr::include_graphics("figures/proportion_barcoded.pdf")
```


## Recommendations

Based on the results of this study, we recommend a multi-level approach for both improving the quality of existing data and for the efficient collection of new data. Our recommendations extends those formulated by Duffy et al. [@Duffy2013], and are informed by the results of our analyses.

### Improve existing data quality and access

Duffy et al. [@Duffy2013] suggested that resources should be invested in the digitization of historical biodiversity to study changes in biodiversity through time in response to climate change, fishing pressure, and pollution. Here, we extend this recommendation to also include improving the quality of the data that is already digitized.

With data aggregation of natural history collections through initiatives like iDigBio, and the compilation of a nearly complete register of marine species with WoRMS, in combination with the development of algorithms for text analysis, it is increasingly easier to validate and spell check the scientific names of the identifications of digitized museum specimens. Linking species names to identifiers in taxonomic databases would facilitate data linking, specimen discoverability, and data usability. 

Similarly, a significant fraction of museum records have missing or inaccurate geographic information associated with them. Geographic information might be available in other forms, but needs to be digitized. While tools have been developed to convert names of locales into GPS coordinates on land [@rios2010geolocate], similar approaches are less-developed for the sea. 

For both the spelling of scientific names, and the geographic information, computing tools can be developed to detect issues and provide solutions. There are opportunities to develop computing services that use publicly available data to flag and fix these issues directly from collection databases records (e.g. see [@Otegui2016] for vertebrate records). Such services, however, require that resources be made available to museum collections in order to address these issues directly in collection databases. 

Increasing the computing literacy and teaching "data wrangling" skills to staff of museums in order for them to manipulate and use the data from their collection database is equally critical. As data across institutions are incorporated and pooled into large biodiversity information systems, it is crucial that data standards such as Darwin Core are embraced by the community. As more people know how to analyze data, and in turn, use the data for their research, inaccuracies can be fixed and overall data quality can improve. 

Another venue to improve the quality of the data in biodiversity information systems, is to develop better incentives from journal editors and funding agencies for scientists to publish their occurrence records in databases alongside their papers as it is done for genetic data with GenBank [@Huang2011e].


### Collect better data faster and more cost effectively

This study highlights that we have too little data about too few species of marine invertebrates. We recommend that additional coordinated field expeditions be organized to sample more comprehensively the marine environment. The relationship between sampling effort and diversity from museum collections makes clear that any additional samples will lead to improving our knowledge of species ranges, and will lead to the discovery of many new species. Concerted and collaborative efforts are needed to share expertise and limited resources. Biodiversity surveys that target all animal phyla can be very effective in documenting biodiversity. For instance, one of us (GP) participated in the summer 2017 in a an inventory of Kaneohe Bay, Hawaii, where ~2,400 records representing ~1,300 species were collected in 11 days of field work. While diversity is higher in Hawaii than in the continental US, this effort led to the collection of a number of species that represent a few percents of the total diversity of the US EEZ marine invertebrate fauna. It demonstrates that documenting the marine fauna of the US is doable.

Museum specimens are critical for validating the occurrence of species [@Rocha2014], especially for marine invertebrates for which species limits based on morphology have been known to be challenged by other lines of evidence including molecular data [@Knowlton2000]. Samples collected during biodiversity surveys need to be photographed to document live appearance, information that would otherwise be lost during the preservation process. This has been shown to be particularly important to validate species limits in cryptic complexes. We also need to capture metadata associated with specimens to be able to understand better the physical characteristics of the habitats and the life histories of these species. Specimens need to be sub-sampled to create material suitable for genomic analysis. Parts of data acquisition associated with specimens and their biota can be automated and captured digitally directly in the field, speeding up their ingestion in database collections.

Integrating metabarcoding as part of the data collection conducted in biodiversity surveys is a promising approach to screen an habitat for its potential to lead new discoveries. For instance, by analyzing metabarcoding samples that contain planktonic larvae from a given habitat, we could get an indication of the taxonomic groups that would benefit most from additional sampling effort, or provide an estimate for the proportion of species that has not been sampled yet. Additionally, data generated by metabarcoding efforts can be re-interpreted as more DNA barcodes associated with correctly identified specimens become available. Building the library of DNA barcodes in parallel with metabarcoding efforts ensures that database matches correspond to actual specimens deposited in natural history collections, ruling out possible artifacts of the methods and permits resolution of specimen identification conflicts.  With such approaches, the geographic distribution of species can build on and be linked to existing records, species and their natural history, rather than being limited to matching DNA sequences across studies.

Metabarcoding also offers to be a promising technology to monitor changes in communities and for understanding the functioning of ecosystems. For instance, despite our limited knowledge of the diversity of communities in the meiofauna, metabarcoding has been successfully used to detect and documents changes in species assemblages following the Deepwater Horizon oil spill [@Bik2012]. Metabarcoding has also revealed important differences in the diet of three related species of fishes that co-occur at small spatial scales, providing a much more detailed understanding of trophic relations [@Leray2015b].

Assembling species lists is a necessary but not sufficient step in our quest to understand biodiversity. We also need to understand better the geographic distribution of biodiversity, how the abundance of species varies in space and time, and species' roles in the functioning of ecosystems. In order to predict the impact of climate change and other anthropogenic perturbations on species distributions, we need to collect additional occurrences for species as well as the characteristics of their habitats. Among the challenges of this enterprise, we need to be able to convey taxonomic concepts for species that, for the most part, have not formally been named yet. Developing robust identifiers that can be shared across biodiversity information systems, and anchored in results from DNA barcoding or other molecular techniques, is needed to collaboratively build our knowledge of species distributions.


## Conclusion

Given the urgency of the biodiversity crisis, we need to develop data-driven initiatives to document biodiversity more efficiently, and build on our existing knowledge to collect data that will improve our understanding of marine systems.

To make biodiversity science and conservation decisions more data driven, we need to ensure that biodiversity data is accurate, comprehensive, freely available, and that the datasets are linked. DNA sequences are an early test case on how to link biodiversity data across repositories. These initial efforts have revealed the lack of infrastructure for using data formats specifically designed for this task, the lack of ontologies available for biodiversity data, and the fragility of the identifiers [@Stucky2014; @Walls2014; @Guralnick2014]. As new types of data will be collected, added, and linked across repositories, the community needs to develop modern and robust ways to integrate these heterogeneous biodiversity data sources [@Guralnick2015; @Deck2015].

Digitization efforts are catching up on already fragmented and incomplete records. Even if technological advances can greatly speed up the documentation of species and their distributions, taxonomists and experts are needed  to collect, sort, process, and identify samples. Taxonomists play a crucial role in updating and synthesizing species-level knowledge that can be used to provide accurate identifications for DNA barcoded samples. Taxonomic expertise within the US is dwindling, and there are now only a handful of US natural history museums with active collections of marine invertebrates. This lack of resources is not sustainable in the face of the challenges and discoveries that marine ecosystems have to offer.

Crowd-sourced data and citizen science initiatives have been proposed to augment data about species abundance and distributions [@Pimm2014]. Beyond the application to near-shore habitats on easily recognizable species [@Delaney2008], large-scale efforts are currently not possible for marine ecosystems given the specialized equipment needed to sample most habitats and the expertise required to identify species. There are however opportunities for citizen scientists to help curate biodiversity data, while educating and engaging the public with marine conservation issues. For instance citizen science has been used successfully to classify plankton images [@Robinson2017], and to transcribe hand-written museum labels [@Hill2012].

Here, we reported results from the US where there are at least two centuries of data on biodiversity exploration data and high-levels of digitization of museum collections. Many other parts of the world need additional attention, especially in some developing countries where environmental pressure on ecosystems is even higher than in the US.


## Methods

The list of species for the Gulf of Mexico was obtained  from http://e-gulf.org/ [@egulf]. This database includes the species listed in a comprehensive biotic inventory of the Gulf of Mexico [@Felder2009].

The list of species for the Pacific Northwest was downloaded from:  https://www.eopugetsound.org/species/custom-lists/307 (accessed on July 5th, 2017). This is a digital version of the species included in [@Kozloff1996].


<!-- Our goals were (1) to compare  the species lists for these geographic areas assembled by experts with the records found in databases; (2) to assess the proportion of species included in the expert lists that have records in public databases. -->

<!-- First, we compared the faunistic lists from the Gulf of Mexico and the Pacific Northwest with records in public databases. we queried the iDigBio (https://portal.idigbio.org) and OBIS (http://www.iobis.org/) databases for all records they included in the same geographic boundaries as the lists. We filtered the results to only include the same higher taxonomic level taxa as in the faunistic lists, and kept only records for which the species could be matched to a species name in WoRMS. For the Gulf of Mexico, we retrieved -->

Species occurence data and museum records were extracted from OBIS (Ocean Biogeographic Information System) and iDigBio (Integrated Digitized Biocollections) respectively. OBIS gathers and integrates datasets about occurrences of marine species around the world provided by over 500 institutions worldwide. iDigBio provides access to the data associated with museum records that have been digitized and made publicly available. While OBIS records can be based on observation, there is a physical specimen deposited in a collection associated with each iDigBio record. 

For geographic searches in OBIS and iDigBio, we obtained the boundaries of the Exclusive Economic Zone for the United States from marineregions.org (mrgid:8456) [@mregions]. For the Pacific Northwest, Kozloff did not specify the exact geographic extent covered by his book. We inferred a polygon that encompass part of the Oregon coast to the northern parts of Vancouver island. For the Gulf of Mexico, we used the same geographic area as the one defined in [@Tunnell2009] by modifying manually the eastern boundaries of the shape file for the Gulf of Mexico provided by marineregions.org (mrgid:4288). These geographic areas were split into 0.2° $\times$ 0.2° quadrats that were used to query the web API of iDigBio and OBIS using the R packages ridigbio [@ridigbio-0.3.5] and robis [@robis-0.1.8] respectively. The resulting occurrences were then clipped to only include records that fell within the EEZ boundaries.

We opted to limit our queries to records with geographic coordinates instead of relying on the "Country" fields of these databases. Tests showed that these queries would return numerous records outside the geographic range of interest for this study (for instance records collected in Hawaii, Alaska, or the US Virgin Islands), while not taking into account records falling within the US EEZ where the "Country" field contained "Atlantic Ocean" for instance.

All species names obtained from all sources were checked against the World Registry of Marine Species (WoRMS). Species that could not be matched in WoRMS, and species listed as non-marine in WoRMS were not included in subsequent analyses. For all names that were listed in WoRMS, the current accepted name was captured and used in all analyses. This ensured that the same species listed under different synonyms was only counted once. The R package worrms [@worrms-0.1.0] was used to query the WoRMS database.

To generate the combined list of records from iDigBio and OBIS, we removed putative duplicate based on the following criteria: same phylum, same accepted species name from WoRMS, same geographic coordinates (rounded to the 4th decimal), and same date of collection. When the date was missing, a single record per species with identical coordinates is included.

To infer the number of species along the latitudinal gradient (Fig. \@ref(fig:latitudinal-gradient)), we estimated the species ranges by using the southernmost and northernmost observations for each species on the Atlantic and Pacific coast separately. Records for the Gulf of Mexico were not included in this analysis.

To estimate the proportion of the marine invertebrate diversity for which DNA barcodes are available, we queried BOLD (Barcode of Life Datasystem) for all species listed in the regional lists for the Gulf of Mexico, the Pacific Northwest, as well as all the records for marine invertebrates that occur in the EEZ of the US found in iDigBio.

To assess the proportion of species for which DNA barcodes are available, we queried the Barcode of Life Datasystem (BOLD) using the R package bold [@bold-0.4.0]. We used the accepted names of species in the other lists, as well as synonyms (as listed in WoRMS) when accepted names did not return any records.

This study is fully reproducible and the code needed to perform the analyses included here is available from: https://github.com/fmichonneau/sok-marine-biodiversity, and archived on Zenodo (doi:xxx.xxx.xxxx). The data downloaded from the databases at the time of analysis are archived on Zenodo (doi:xxx.xxx.xxx).


## Acknowledgments

We thank WoRMS, BOLD, iDigBio, OBIS, MarineRegions, NCBI, and all other data providers that aim to make more accessible biodiversity data. We would also like to acknowledge the efforts of rOpenSci (https://ropensci.org) for developping tools to make these data sources available in R. We are grateful to Julie Allen, Robert Lasley, Kevin Love, and Pamela Soltis for providing suggestions that improved the manuscript.


## References

<div id="refs"></div>


\newpage

# Supplementary material


\renewcommand{\thefigure}{S\arabic{figure}}

\setcounter{figure}{0}


```{r gom-maps, fig.cap="Spatial distribution of the number of samples (A) and species richness (B) for marine invertebrates based on records retrieved from iDigBio and OBIS for the Gulf of Mexico. Geographic extent represented in light gray."}
knitr::include_graphics("figures/combined_gom_sampling.pdf")
```

```{r pnw-maps, fig.cap="Spatial distribution of the number of samples (A) and species richness (B) for marine invertebrates based on records retrieved from iDigBio and OBIS for the Pacific Northwest. Geographic extent represented in light gray."}
knitr::include_graphics("figures/combined_pnw_sampling.pdf")
```

```{r idigbio-usa-maps, fig.cap="Spatial distribution of the number of samples (A), and species richness (B) for marine invertebrates based on records retrieved from iDigBio within the limits of the Exclusive Economic Zone for the contiental USA. Geographic extent represented in light gray."}
knitr::include_graphics("figures/idigbio_map_sampling.pdf")
```

```{r obis-usa-maps, fig.cap="Spatial distribution of the number of samples (A), and species richness (B) for marine invertebrates based on records retrieved from OBIS within the limits of the Exclusive Economic Zone for the contiental USA. Geographic extent represented in light gray."}
knitr::include_graphics("figures/obis_map_sampling.pdf")
```

```{r rank-abundance, fig.cap="Rank abundance plots for most represented phyla based on the number of samples (A), and the number of cells occupied in map of Fig. \\@ref(fig:map-diversity-sampling) (B)"}
knitr::include_graphics("figures/plot_rank_abundance.pdf")
```


* list of marine invertebrate species from the continental USA: `list_species_us.csv`
* list of marine invertebrate species from the Gulf of Mexico: `list_species_gom.csv`
* list of marine invertebrate species from the Pacific Northwest: `list_species_pnw.csv`
